src/javascript/app/base/header.js

```diff
@@ -285,16 +286,20 @@ const Header = (() => {
 
             const buildMessage = (string, path, hash = '') => template(string, [`<a href="${Url.urlFor(path)}${hash}">`, '</a>']);
             const hasStatus = (string) => status.findIndex(s => s === string) < 0 ? Boolean(false) : Boolean(true);
+            const hasVerification = (string) => needs_verification.findIndex(s => s === string) < 0
+                ? Boolean(false) : Boolean(true);
```

Comment:
can't we change it to `const hasVerification = (string) => needs_verification.findIndex(s => s === string) >= 0;`? it returns true if index found else returns false, and no need to have extra if condition